barustenberg/src/plonk/proof_system/commitment_scheme.rs

```diff
@@ -215,7 +215,7 @@ impl<S: Settings<Hasher = H, Field = Fr, Group = G1Affine>, H: BarretenHasher> C
     fn generic_batch_open(
         &mut self,
         src: &[Fr],
-        dest: Arc<RwLock<Polynomial<Fr>>>,
+        dest: &mut Arc<RwLock<Polynomial<Fr>>>,
```

Comment:
why......... this is wrong for sure

Reply:
Yeah... This is just wrong. Don't know what I was thinking. I fixed it.